platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -1087,6 +1088,17 @@ private AddressBook getAddressBook() {
         return addressBook;
     }
 
+    /**
+     * Checks if the event has the same index and the one provided.
```

Comment:
Was this meant to say "_as_ the one provided" ?

That being said, I don't think this doc is a great description of what's being checked. The method is checking whether the *creator* of the event has a certain index in the address book. I think the parameter docs could also be updated for clarity

Reply:
updated javadoc for clarity.